leftwm/src/bin/leftwm-check.rs

```diff
@@ -10,41 +10,21 @@ use std::path::Path;
 use std::path::PathBuf;
 use xdg::BaseDirectories;
 
-const INPUT_ARG: &str = "input";
-const VERBOSE_ARG: &str = "verbose";
-const MIGRATE_ARG: &str = "migrate";
-
 #[tokio::main]
 async fn main() -> Result<()> {
     let matches = command!("LeftWM Check")
         .author("Lex Childs <lex.childs@gmail.com>")
         .version(env!("CARGO_PKG_VERSION"))
         .about("checks syntax of the configuration file")
-        .arg(
-            Arg::new(INPUT_ARG)
-                .value_name("INPUT")
-                .help("Sets the input file to use. Uses first in PATH otherwise.")
-        )
-        .arg(
-            Arg::new(VERBOSE_ARG)
-                .short('v')
-                .long("verbose")
-                .help("Outputs received configuration file.")
-        )
-        .arg(
-            Arg::new(MIGRATE_ARG)
-                .short('m')
-                .long("migrate-toml-to-ron")
-                .help("Migrates an exesting `toml` based config to a `ron` based one.\nKeeps the old file for reference, please delete it manually.")
-        )
+        .args(&[
+            arg!(-v --verbose "Outputs received configuration file."),
```

Comment:
Couldn't we also do:
```suggestion
            arg!(verbose: -v "Outputs received configuration file."),
```

Reply:
Yes, but then you could only use -v and not --verbose on the command line.

Reply:
I see, I thought, the syntax of `migrate` would result in the three options `-m, --migrate, --migrate-toml-to-ron`, but that obviously was a missconception then.